chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/presenter/CertifiedProductXmlPresenter.java

```diff
@@ -23,24 +28,35 @@
 
     @Override
     public int presentAsFile(final File file, final CertifiedProductDownloadResponse cpList) {
-        int numRecords = 0;
-        FileOutputStream os = null;
         try {
-            os = new FileOutputStream(file);
-            Jaxb2Marshaller marshaller = new Jaxb2Marshaller();
-            marshaller.setClassesToBeBound(cpList.getClass());
-            marshaller.marshal(cpList, new StreamResult(os));
-            numRecords = (cpList.getListings() == null ? 0 : cpList.getListings().size());
-        } catch (final FileNotFoundException ex) {
-            LOGGER.error("file not found " + file);
-        } finally {
-            if (os != null) {
-                try {
-                    os.close();
-                } catch (final IOException ignore) {
+            Writer writer = null;
+            XMLStreamWriter streamWriter = null;
+            try {
+                XMLOutputFactory factory = XMLOutputFactory.newInstance();
+                writer = new OutputStreamWriter(new FileOutputStream(file), StandardCharsets.UTF_8);
+                //streamWriter = factory.createXMLStreamWriter(writer);
+                streamWriter = new IndentingXMLStreamWriter(factory.createXMLStreamWriter(writer));
```

Comment:
Does an "IndentingXMLStreamWriter" always indent? I thought we weren't going to do pretty printing?

Reply:
Addressed